src/js/components/Button/Button.js

```diff
@@ -271,6 +274,19 @@ const Button = forwardRef(
       return result;
     }, [active, disabled, kind, kindObj, plain, selected]);
 
+    if (skeleton) {
+       return (
+        <Skeleton
+          ref={ref}
+          height={theme.text[size || 'medium']?.height || size}
+          a11yTitle={a11yTitle}
+          { ...rest }
```

Comment:
It looks like there's extra spaces here that the linter/prettier usually pulls out.

Reply:
same with some other spots in the code.